man/Makefile.am

```diff
@@ -101,7 +101,24 @@ $(MANS): doxyfile-knet.stamp
 
 doxyfile-knet.stamp: $(builddir)/doxyxml Doxyfile-knet $(top_srcdir)/libknet/libknet.h
 	$(DOXYGEN) Doxyfile-knet
-	$(builddir)/doxyxml -m -P -o $(builddir) -s 3 -p @PACKAGE_NAME@ -H "Kronosnet Programmer's Manual" -d $(builddir)/xml-knet/ libknet_8h.xml
+	if test -d $(top_srcdir)/.git; then \
+		echo "getting git version" && \
+		mandate=`git log -1 --pretty=format:%ci | $(AWK) '{print $$1}'`; \
```

Comment:
Shouldn't we convert to a fixed time zone here?
Not very important, we just have to choose what to be consistent with: earlier values or the commit time zone...